### PR TITLE
Quizmaster must be able to create a quiz

### PR DESCRIPTION
**Pull request by Blokkinen**
PT Story: https://www.pivotaltracker.com/story/show/133242723

Changes proposed in this pull request:

- Add view for "create a quiz"
- Add a form and button to create the quiz
- Acceptence tests

What I have learned working on this feature: 

-  A lot of things!

Screenshots: 

skarmavbild 2016-11-01 kl 16 18 53

Ready for review!

### CODE REVIEW

File: app/controllers/quizmaster/quizzes_controller.rb [hunk 1]
```diff
@@ -79,4 +92,8 @@ def get_winner
     winner
   end
 
+  def quiz_params
+    params.permit(:name)
+  end
+
 end
```
Comment (AmberWilkie) [inline app/controllers/quizmaster/quizzes_controller.rb]: Ok while you're at it clean up the white space in this controller. (All the extra blank lines)

File: features/quizmaster/create_quiz.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Background:
+  Given there is a quizmaster with email "viktoria@quizmaster.com"
+  And I am logged in as "viktoria@quizmaster.com"
+
+
+Scenario: I create a quiz
+  Given I am on the quizmaster "Create quiz" page
+  When I fill in "Name" with "Team Craft Academy"
+  And I click the "Create" button
+  Then "viktoria@quizmaster.com" should have 1 quiz
```
Comment (AmberWilkie) [inline features/quizmaster/create_quiz.feature]: What about sad path?

File: features/step_definitions/basic_steps.rb [hunk 3]
```diff
@@ -34,6 +34,10 @@
   end
 end
 
+Given(/^I am on the quizmaster Add questions page for "([^"]*)"$/) do |quiz|
+  quiz = Quiz.find_by(name: name)
+  visit quizmaster_add_questions_path(quiz)
+end
```
Comment (AmberWilkie) [inline features/step_definitions/basic_steps.rb]: This test goes to a route that doesn't exist, right? What is this add_questions path? Is this step not being used? If so, lose it.

Comment (Blokkinen) [inline features/step_definitions/basic_steps.rb]: It is used for "Then "Amber Rocks!" should have "1" correct answer" in the "Create quiz" feature

**Pull request merged**